assets/js/OutputPane.js

```diff
@@ -168,6 +195,8 @@ export default class OutputPane extends React.Component {
         resizing={this.state.resizing}
         api={this.props.api}
         setBusySpinner={this.setBusySpinner}
+        sortColumn={moduleIsSort ? currentModule.parameter_vals[0].value : undefined}
```

Comment:
Why parameter_vals[0] and [2]? You need to get them by id_name so that this code still works if parameters are re-ordered.